dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -652,6 +695,38 @@ Simulation::importCurvesRequest() {
   curvesCollection_ = CurvesCollectionFactory::copyInstance(importer.importFromFile(curvesInputFile_));
 }
 
+void
+Simulation::importFinalStateValuesRequest() {
+  // Obtain Final State Values definitions from input file
+  final_state_values::XmlImporter importer;
+  auto finalStateValuesCollection =
+      final_state_values::FinalStateValuesCollectionFactory::copyInstance(
+          importer.importFromFile(finalStateValuesInputFile_));
+
+  // Final State Values and Curve points will be recorded during simulation in the same data structures
+  // Here we merge the Final State Values definitions with the existing Curves definitions
+
+  // A map for existing Curves is built so we can locate them fast and update them from the Final State Values
+  // A Curve is identified by the pair model name, variable name
+  auto curvesMap = boost::unordered_map<std::pair<std::string, std::string>, boost::shared_ptr<curves::Curve>>();
```

Comment:
This syntax makes a copy of an empty object. No impacts on performance but it is better to declare the object normally.
Besides, this doesn't compile in cpp98: ">>" => "> >"

Reply:
Declaration adjusted

Reply:
removed `auto` declarations